packages/console-types/src/types/services.ts

```diff
@@ -520,6 +520,7 @@ export const container = {
           label: { type: 'string' },
           enableIf: { type: 'string' },
           targetSection: { type: 'string' },
+          hideViewPrefix: { type: 'boolean' },
```

Comment:
```suggestion
          hidePrefix: { type: 'boolean' },
```

As defined in RFC-55